discord/ui/view.py

```diff
@@ -341,7 +345,10 @@ async def on_timeout(self) -> None:
 
         A callback that is called when a view's timeout elapses without being explicitly stopped.
         """
-        pass
+        if self.disable_on_timeout:
+            if self._message:
+                self.disable_all_items()
+                await self._message.edit(view=self)
```

Comment:
I don't really think it's a good idea to have the lib edit the message by default

Reply:
It doesn't. The disable_on_timeout kwarg defaults to False.